internal/storage/ledger/balances.go

```diff
@@ -14,13 +14,15 @@ import (
 	ledgercontroller "github.com/formancehq/ledger/internal/controller/ledger"
 )
 
+const fillAccountsVolumeHistoryMigration = 21
```

Comment:
THis change is not related to this PR, but it avoid the fallback to the old way when upgrading the ledger.